### ISSUE TITLE
[Feature request] Add More Discord Actions

### ISSUE THREAD

**Issue by J-Daniel-1** [Twentysix26/x26-Cogs#8]
**Cog**
- [x] Defender
- [ ] Index
- [ ] Sbansync

**Describe the feature you'd like to see**
Add more Discord action actions like `mute-user` so I can create a modlog with a specified reason. 

`send-mod-log` does **not** add a reason to manually adding the muted role to a user.

**Comment by Twentysix26** [comment]
I'd like to support core muting functionalities but at the moment there's no API for cogs to interface with. I'll hold off until then

**Comment by Twentysix26** [comment]
FYI you can achieve the same end result with `issue-command` now. This will stay open until there is proper support though.

**Comment by Jac0bDan13L** [comment]
With the Mutes cog, does that open up the API or no, still?

**Comment by Twentysix26** [comment]
> With the Mutes cog, does that open up the API or no, still?

No. You will need to have it loaded if you want to make use of it through `issue-command` though.

**Comment by DoctorDinosaur** [comment]
Kinda superceded by #64 which has been added to dev branch already.
